.flake8

```diff
@@ -2,3 +2,7 @@
 
 # Let's not overcomplicate the code:
 max-complexity = 10
+
+# Don't even try to analyze these:
+extend-exclude =
+  *migrations*
```

Comment:
Use a full path, not a wildcard matching any file with `migrations` in the middle of its name.

Reply:
Marking as resolved.